corehq/apps/app_manager/models.py

```diff
@@ -2609,6 +2613,15 @@ def get_case_errors(self, needs_case_type, needs_case_detail, needs_referral_det
             for error in errors:
                 yield error
 
+    def validate_for_build(self):
```

Comment:
This hasn't been tested as I'm not sure how to make a careplan app. Can anyone point me in the right direction?